archivy/models.py

```diff
@@ -92,7 +93,15 @@ def process_bookmark_url(self):
             return None
 
         try:
-            url_request = requests.get(self.url)
+            try:
+                lc = locale.getdefaultlocale()[0].replace('_', '-')
```

Comment:
Does this work for every locale (replacing `_` with `-`)?

Reply:
I hope it works, is there any case it doesnt?

Reply:
I'm not sure, I don't know much about locales and how it works with web requests, so I was asking since you opened the PR.

I just want to make sure the code we merge in works well :)